File: structy/introduction/max-value.js
/* 
Max Value
https://www.structy.net/problems/max-value
13 August 2023
David Pratt
*/

const maxValue = (nums) => {
  let max = -Infinity;
  for(let val of nums) if(val > max) max = val;
  return max;
};

File: structy/introduction/max-value.py
'''
Max Value
12 November 2023
https://structy.net/problems/max-value
David Pratt
'''

def max_value(nums):
  max = float('-inf')
  for i in range(len(nums)):
    if(nums[i] > max):
      max = nums[i]
  return max
